Exiled.API/Extensions/ItemExtensions.cs

```diff
@@ -194,10 +194,18 @@ public static IEnumerable<AttachmentIdentifier> GetAttachmentIdentifiers(this It
                 throw new System.ArgumentException("The attachments code can't be less than the item's base code.");
             }
 
+            IEnumerable<AttachmentIdentifier> attachmentIdentifiers = Firearm.AvailableAttachments[type];
+            IEnumerable<uint> sources = attachmentIdentifiers.Select(attId => attId.Code);
```

Comment:
Since you're using .Count() and multple .ElementAt()'s would it not be more efficient to make this a uint[] sources = attachmentIdentifiers.Select(attId => attId.Code).ToArray() so you aren't enumerating this multiple times, and then just use sources.Length and sources[i] ?

Reply:
I mean, the performance is pretty the same. But I'll change it.